pkg/config/config.go

```diff
@@ -79,6 +82,11 @@ func (cfg *Config) BindFlags() {
 		"",
 		"The AWS Region in which the service controller will create its resources",
 	)
+	flag.StringVar(
+		&cfg.EndpointURL, flagAWSEndpointURL,
+		"",
+		"The AWS endpoint URL the service controller will use to create its resources",
```

Comment:
Please add some text that says this is only used to override the default behaviour of aws-sdk-go that constructs endpoint URLs automatically based on service and region.